### ISSUE TITLE
Avoid https for mods to in playlists?

### ISSUE THREAD

**Issue by kowoba** [koobo/HippoPlayer#21]
It looks like the mods that show up in shared playlists are mostly (if not all) only available over https, and hence require AmiSSL and do not work with 68000/010. Would it be possible use http instead? And if not… allow users to specify alternative “get” command for fetching URLs, so we can fix the ssl-offloading ourselves? I understand that Hippo is somehow linked to UHC-Tools aget, but it doesn’t apparar to use the aget binary directly?

**Comment by kowoba** [comment]
AmiGemini comes with a proxy with ssl offloading for all relevant protocols, maybe the best would be if aget could support that proxy as well… I can try connecting Patrik and Karl :)

**Comment by koobo** [comment]
This might actually be a problem in the shared lists themselves, since they all should work with "http", with the possible exception of _modules.pl_. Will check.

Hippo does actually use the "aget" binary directly as if running it from the shell.

**Comment by koobo** [comment]
The lists should be mostly without https now.

**Comment by kowoba** [comment]
Hm, I still get requester about "https:// requires amisslmaster.library v4" whenever I try to "play" any of the Shared lists. Note, I am talking specifically about the "Shared lists" (second last in the menu of AmigaRemix, Aminet, HVSC etc) - I do get the list (starting with "8GB's Amiga DJ Mix Vol. 2" on top), but when trying to load any of these playlist entries, I get the error about https.  I can understand that individual mods in a playlist may be behind https, but the problem here is accessing the lists themselves.

I have not investigated thoroughly, but I some similarities between this list of playlists and the content of .prg files in the "Playlists" drawer on https://hippoplayer.se  :)

I quickly tried to replace UHC "aget" with a shell script wrapper (with file flag +s), with same version string as original aget and accepting same options, in attempt to catch the specific URLs it tries to load, and to offload SSL handling to a proxy, but it fails to work, perhaps HippoPlayer expects aget to be a binary... SnoopDOS wasn't so helpful :)

**Comment by koobo** [comment]
You're right there's still an issue with getting the lists as you said, will check it.  And yes you're also right about the "Playlists" drawer, it is the same content that is available in the "Shared lists" search :-)

**Comment by koobo** [comment]
The issue should now be solved, can you give it a try?
